packages/terra-search-field/tests/wdio/search-field-spec.js

```diff
@@ -0,0 +1,252 @@
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/default-search-field'));
+
+    Terra.should.matchScreenshot('empty');
+
+    it('should enter a search term', () => {
+      browser.setValue('input', 'Lorem');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('with text');
+
+    it('should scroll text that is too long', () => {
+      browser.addValue('input', ' ipsum dolor sit amet');
+    });
+
+    Terra.should.matchScreenshot('scrolled text');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+  });
+
+  describe('Block', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-block'));
+
+    Terra.should.matchScreenshot('empty');
+
+    it('should enter a search term', () => {
+      browser.setValue('input', 'Lorem');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('with text');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+  });
+
+  describe('Disabled', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-disabled'));
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible({ rules: ignoredA11y });
+
+    it('should not accept keyboard input', () => {
+      expect(browser.setValue.bind(browser, 'input', 'Lorem')).to
+        .throw('Element is not currently interactable and may not be manipulated');
+    });
+
+    it('should not accept clicks', () => {
+      expect(browser.click.bind(browser, '[class*="button"]')).to.throw(Error);
+    });
+  });
+
+  describe('With Placeholder', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-with-placeholder'));
+
+    Terra.should.matchScreenshot('placeholder');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+
+    it('should enter a search term', () => {
+      browser.setValue('input', 'Lorem');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('with text');
+  });
+
+  describe('With Default Value', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-default-value'));
+
+    Terra.should.matchScreenshot('default value');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+
+    it('should enter a new search term', () => {
+      browser.click('input');
+      for (let i = 0; i < 7; i += 1) {
+        browser.keys('Backspace');
```

Comment:
I would expect the backspace to be in an array like ['Backspace'] Otherwise, I think the test is writing all those values to the input, and getting cleared when addValue is called.

Reply:
`addValue` appends to the input but `setValue` would overwrite the input. I decided to use `browser.keys('Backspace');` to test to be sure that the default value could actually be cleared.

Reply:
We should still be able to use an array like Jake suggested if we update to use 

```
 browser.keys(['Backspace', 'Backspace', 'Backspace',...]);
```
Though I have no preference on which approach is cleaner. 

In regard to check if the default value was actually cleared, there is no assertion or screenshot taken to ensure the value is actually empty before we use `addValue('input', 'Lorem')`